source/blog/2015-02-03-rspec-3-2-has-been-released.md

```diff
@@ -0,0 +1,656 @@
+---
+title: RSpec 3.2 has been released!
+author: Myron Marston
+published: false
+---
+
+(TODO: consider writing a new intro.)
+
+RSpec 3.2 has just been released! Given our commitment to
+[semantic versioning](http://semver.org/), this should be a trivial
+upgrade for anyone already using RSpec 3.0 or 3.1, but if we did introduce
+any regressions, please let us know, and we'll get a patch release
+out with a fix ASAP.
+
+RSpec continues to be a community-driven project with contributors
+from all over the world. This release includes 647 commits from 190
+pull requests from 47 different contributors.
```

Comment:
Actually the numbers are even higher (see stats below).  This is copied in from the 3.1 release post.  I plan to update it and/or rewrite this section.